pkgs/applications/networking/browsers/palemoon/default.nix

```diff
@@ -155,7 +152,7 @@ stdenv.mkDerivation rec {
 
     # Remove unneeded SDK data from installation
     # https://forum.palemoon.org/viewtopic.php?f=37&t=26796&p=214676#p214729
-    rm -rf $out/{include,share/idl,lib/palemoon-devel-${version}}
+    rm -r $out/{include,share/idl,lib/palemoon-devel-${version}}
```

Comment:
This was temporarily not a problem anymore during the short lifespan of v30, but it seems to have resurfaced with v31. Should prolly tip the devs off to this back-and-forth, not the time rn though.